unified-installer/astra-unified-installer.sh

```diff
@@ -357,17 +373,20 @@ load_config_from_file_if_given() {
 
     # Set config file values first
     ingest_config_string "$(cat "$config_file")"
-    set_log_level
 
     # Check if api token was populated after sourcing config file
     if [ "$api_token" != "$ASTRA_API_TOKEN" ]; then
-        logwarn "$token_warning"
```

Comment:
This log was being called after we ingested the config file, but before we apply values given through the command line. This meant that, if the config file had `LOG_LEVEL=debug` but the command line had `LOG_LEVEL=info`, you would see this warning with the `|WARN |` prefix even though it shouldn't show with log level **info**.